src/main/scala/gestalt/dotty/DottyToolbox.scala

```diff
@@ -1189,6 +1191,16 @@ class Toolbox(enclosingPosition: Position)(implicit ctx: Context) extends Tbox {
     def appliedTo(tp: Type, args: List[Type]): Type = new TypeApplications(tp).appliedTo(args)(ctx)
 
     def toTree(tp: Type): tpd.Tree = t.TypeTree(tp)
+
+    /** Infer an implicit instance of the given type */
+    def infer(tp: Type): Option[tpd.Tree] = {
```

Comment:
If it infers and implicit instance shouldn't the method be called `inferImplicitInstance` or `inferImplicit`?